src/dirigera/devices/outlet.py

```diff
@@ -0,0 +1,80 @@
+from dataclasses import dataclass
+from typing import Any, Optional, Dict
+
+from .device import Device, StartupEnum
+
+from ..hub.abstract_smart_home_hub import AbstractSmartHomeHub
+
+# Quirks:
+# TRADFRI control outlet / IKEA of Sweden:
+# device has lightLevel attribute and canReceive capability, neither of
+# them affect the state of relay.
+
+@dataclass
+class Outlet(Device):
+    dirigera_client: AbstractSmartHomeHub
+    is_reachable: bool
+    is_on: bool
+    startup_on_off: Optional[StartupEnum]
+
+    def refresh(self) -> None:
+        data = self.dirigera_client.get(route=f"/devices/{self.device_id}")
+        attributes: Dict[str, Any] = data["attributes"]
+        self.device_id = data["id"]
+        self.is_reachable = data["isReachable"]
+        self.custom_name = attributes["customName"]
+        self.is_on = attributes["isOn"]
+        self.startup_on_off = StartupEnum(attributes.get("startupOnOff"))
```

Comment:
If startup_on_off is truly optional then make it similar to the changes you made in light.py
```python
        if "startupOnOff" in attributes:
            startup_on_off = StartupEnum(attributes.get("startupOnOff"))
        else:
            startup_on_off = None
```

Reply:
Done. This optionality is purely based on [Pull request 2](https://github.com/Leggin/dirigera/pull/2). I don't have access to any 3rd part devices to verify.